index.src.html

```diff
@@ -711,23 +790,45 @@ <h4 id="cors-preflight">CORS preflight</h4>
 
           1.  Return null.
 
-      1.  Let |clientAddressSpace| be |request|'s
-          [=request/policy container=]'s
-          [=policy container/IP address space=].
-
-      1.  If |response|'s [=response/IP address space=] is
+      1.  If |connection|'s [=connection/IP address space=] is
           [=IP address space/less public=] than |clientAddressSpace|, then:
 
               1.  Let |error| be a [=network error=].
 
-              1.  If |request|'s [=request/client=] is a [=secure context=],
-                  then set |error|'s [=response/IP address space=] property to
-                  |response|'s [=response/IP address space=].
+              1.  Set |error|'s [=connection/IP address space=] property to
+                  |connection|'s [=connection/IP address space=].
 
               1.  Return |error|.
 
       1.  Return null.
 
+  1.  The [$HTTP-network fetch$] algorithm is amended as follows:
+
+      1.  Immediately after |request| is defined:
+
+          1.  If all of the following are true
+              - |request|'s [=request/target IP address space=] is not null
+              - |request|'s [=request/policy container=] is not null
+              - |request|'s [=request/target IP address space=] is
+                not [=IP address space/less public=] than |request|'s
+                [=request/policy container=]'s [=policy container/IP address
+                space=]
```

Comment:
Can we check this (and throw if need be) at request construction time, when copying from `RequestInit`? Then we can assert this here (or at the top of main fetch) instead.

Reply:
I think the earliest we can check this is https://fetch.spec.whatwg.org/#ref-for-concept-request-policy-container%E2%91%A2 since that's when the policy container is set.

Reply:
Makes sense!